charts/kubefed/Chart.yaml

```diff
@@ -1,4 +1,4 @@
 apiVersion: v2
 description: KubeFed helm chart
 name: kubefed
-version: 0.0.3
+version: 0.0.4
```

Comment:
Is this change in anticipation of a release? Wouldn't this match the latest released version and better be changed as part of the release process, or a PR preparing for a release?

Reply:
force of habit, apologies.